challenge_7/blogs.yml

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
How do you feel about the trade-offs of committing this YAML file to the repo, versus putting it in the `.gitignore` so that Git doesn’t pay attention to it? What are the downsides either way, e.g. for your experience working on the app, or for someone else checking out the repo for the first time? I don’t think there’s a right answer here, it’s just something to consider. 😄

Reply:
Oh, this is an interesting one! 😁  So back in challenge 6 I had some "seed" blog data that would get pushed into our in-memory data structure so that I could see data right away without having to create blogs via post requests all the time. I kind of like the idea of keeping seeded data, but having Git update the file and continue to track it as I play around locally is a bit of a pain (and if multiple collaborators were working on this project, we wouldn't want to be mucking with this file all the time I imagine!) 

Maybe the way around this is to not commit this file (and to tell Git to ignore it), and if I care about seeding data I can handle that in the Ruby script itself. I think that makes it more obvious why that data is there / where it came from.

Something else I noticed - right now if you _don't_  have anything in the `blogs.yml` , this blows up:
```ruby
    blog_data = store.transaction { store[:blogs] }
    blog_data.each do |element|
      message_body << "<li>"
      message_body << "<strong>Title: #{CGI.escape_html(element.title)}</strong>, Content: #{CGI.escape_html(element.content)}"
      message_body << "</li>"
    end
```
I noticed this when copying the file over to my `challenge_8` directory, but I didn't copy the `blogs.yml` file so I hit ` #<NoMethodError: undefined method `each' for nil:NilClass>` 😅  
Something I will also address before merging this!

Reply:
Yeah, interesting! Is there a way we could make it gracefully set up the store if it doesn’t already exist?

Reply:
I've added https://github.com/adrianna-chang-shopify/learning-project/pull/6/commits/6d4f24ed5458b481c447d059b8dc1823e3f4f5de to:
- set `store[:blogs]` to an empty array if it's nil
- optionally seed some data if `store[:blogs]` is an empty array. This can be commented out though (ie. if a user wanted to start from scratch) and everything should still work 😁

Reply:
Awesome work, and very user-friendly! :shipit: